### ISSUE TITLE
Cut off comment summaries at <hr>

### ISSUE THREAD

**Issue by pkaminski** [integrations/slack#633]
Comments and issue/PR descriptions can get pretty long, and currently the plug-in sends the full text to Slack.  While Slack will collapse it (based on character count?) this can break the text at an unfortunate spot and still show more than is relevant for a summary.

I think it would be nice if the plug-in cut off the content sent to Slack at the first occurrence of a delimiter, most likely `<hr>` since in my experience many people use it to separate the summary from details anyway.

(I'm happy to contribute this feature but want to make sure it's welcome first.)

**Comment by pkaminski** [comment]
@wilhelmklopp could you please take a look at this feature request?  I've got PRs open to implement it if you think it's a good idea.  Thanks.

**Comment by pkaminski** [comment]
This is still relevant, since AFAIK the issue was never addressed.

**Comment by ccahoon** [comment]
I agree, and I’d also be willing to help build or test the change. It’d be really great.

**Comment by pkaminski** [comment]
Hey, it's the annual "let's bump this issue so it doesn't get closed" event!

**Comment by ccahoon** [comment]
I’m still very interested in this feature.
